README.md

```diff
@@ -80,7 +80,7 @@ Once the dependencies installed, you can install the latest release of deeprankc
 pip install deeprankcore
 ```
 
-You can get all the new developments by cloning the repo and installing the code with
+Please note that the PyPi release is currently outdated and we will do a new release very soon. In the meantime, you can get all the new developments by cloning the repo and installing the code with
```

Comment:
it's still unclear to me what the added value of just this statement is. If people can use the package, I don't think we need to say this. If they can't use the package right now, we shouldn't tell them to install.

If we do need to say something about it, I think it needs a bit more explanation. As a non-expert user, if I see this statement, I would have no idea what I should do now.